packages/common/src/types/dbt.ts

```diff
@@ -240,27 +240,36 @@ type DbtMetricFilter = {
     value: string;
 };
 export type DbtMetric = {
+    fqn: string[];
     unique_id: string;
     package_name: string;
     path: string;
     root_path: string;
     original_file_path: string;
-    model: string;
     name: string;
     description: string;
     label: string;
     type: string;
-    timestamp: string | null;
-    filters: DbtMetricFilter[];
+    sql: string;
```

Comment:
is this now Not optional ?

Reply:
Copied the typing from dbt json schema specification.